cloud/terraform/machineactuator.go

```diff
@@ -207,6 +207,10 @@ func (tf *TerraformClient) Create(cluster *clusterv1.Cluster, machine *clusterv1
 		if err != nil {
 			return err
 		}
+		// Save the calico.yaml file.
+		if err := saveFile(podNetworkCNICalico, path.Join("/tmp", "calico.yaml"), 0644); err != nil {
```

Comment:
If the plan is to support multiple CNI providers in the near future, think about how this might make it easier/harder to do so.